hyperspy/drawing/image.py

```diff
@@ -193,6 +194,7 @@ def plot(self):
                     ax=self.ax,
                     units=self.pixel_units,
                     animated=True,
+                    color=self.scalebar_color,
```

Comment:
This line is needed. Without it, the actual ImagePlot does not know what color to use and it defaults to white.